draft-terbu-sd-jwt-vc.md

```diff
@@ -116,7 +116,7 @@ The following are non-selectively disclosable registered JWT claims that SD-JWT-
 * `exp`
     * REQUIRED. The expiry time of the Verifiable Credential after which the proof of the Verifiable Credential is no longer valid.
 * `cnf`
-    * OPTIONAL. The confirmation method can be used to verify the Holder Binding JWT of the disclosed SD-JWT.
+    * OPTIONAL. The confirmation method as defined in [@!RFC7800]. It MAY be used when Cryptographic Holder Binding is required. It MUST contain a JWK as defined in Section 3.2 of [@!RFC7800]. Other representations of the key MUST NOT be used. "kid" (Key ID) member MUST be present in the JWK. Holder Binding JWT in the Combined Format for Presentation MUST be signed by the key identified in this claim.
```

Comment:
I don't understand why this limitation is necessary in this spec?

Reply:
I'd suggest

`It SHOULD be used when Cryptographic holder binding is required` and then `It SHOULD contain a JWK as defined ...` since there may be in the future different kind of cnf objects

regarding `Other representations of the key MUST NOT be used` -> why not? Since together with a JWK there may be a different but equivalent method to represent the same object, in this case the final result must be the same otherwise the attestation should be considered not valid